### DISCUSSION TITLE
max between column and content width

### DISCUSSION THREAD

**Discussion by armandoclasslife** [dbeaver/dbeaver#12075] [Q&A]
Hi guys, I want to twist the twister.

What I would like to have is to fit the longest one between column and content.

I am on Linux. If you fit to the content and the column name is longer than the content, the column name is truncated. So I would like to fit to the longest one between the column name and the content, so that I would always see the column name and the column content entirely.

Is it possible? Feature request?

**Comment by uslss** [comment]
thanks for the report

**Comment by LonwoLonwo** [comment]
Hello @armandoclasslife 
Can you provide more details please?
May be, some pictures?

May be this setting is enabled?
![2021-04-07 18_50_32-DBeaver Enterprise 21 0 2 - reminders](https://user-images.githubusercontent.com/45152336/113895999-31d68200-97d2-11eb-964b-e5ccf2506ac1.png)

**Comment by armandoclasslife** [comment]
Hi,

It is very simple.

Now there are 2 options:
1- fit the column width to the screen (Columns width: fit screen)
2- fit the column width to the content (Columns width: fit values)

So there should be an option where both the column name and the column
content are visible.

So if the column name has 100px of width and its content (the maximum of
all rows) is 150px, the column width should be the maximum between both (on
this case, 150px)

This makes sense because usually we want to see both, the name of the
column and the entire content.

Regards

On Wed, 7 Apr 2021 at 17:51, Anastasiya ***@***.***> wrote:

> Hello @armandoclasslife <https://github.com/armandoclasslife>
> Can you provide more details please?
> May be, some pictures?
>
> May be this setting is enabled?
> [image: 2021-04-07 18_50_32-DBeaver Enterprise 21 0 2 - reminders]
> <https://user-images.githubusercontent.com/45152336/113895999-31d68200-97d2-11eb-964b-e5ccf2506ac1.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-815024515>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS5X5BQVWHNSG6ELQPGNFXTTHR5QRANCNFSM4YD5LQKQ>
> .
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by LonwoLonwo** [comment]
I not sure, that I understand you correctly.

We have some standards for columns width. On default we use _fit screen_ - that you can see, when open table.

You can see all content on default columns in case, when this is some text column, when you are using _fit values_. If we have column name 50 px and content 100px - this column will be 100px. If we have column name 100px and content 50px - this column will be 100px, because we show column names entirely by default.

![2021-04-08_19h28_43](https://user-images.githubusercontent.com/45152336/114062991-ce675580-98a0-11eb-8b61-fa2c19193877.gif)

So, again, we show column names entirely by default, and we have some minimum value from column content, but we do not show column content entirely by default. An if you want to see entirely column names as well as column content - use Columns width: fit values.

But if it does not work (and the setting is disabled, which I mentioned above) - this is bug. And I need to know more details about it.

**Comment by armandoclasslife** [comment]
as you can see on the screenshot, this is not true.

selecting fit values, you don't see the entire column names

regards

On Thu, 8 Apr 2021 at 18:44, Anastasiya ***@***.***> wrote:

> I not sure, that I understand you correctly.
>
> We have some standards for columns width. On default we use *fit screen*
> - that you can see, when open table.
>
> You can see all content on default columns in case, when this is some text
> column, when you are using *fit values*. If we have column name 50 px and
> content 100px - this column will be 100px. If we have column name 100px and
> content 50px - this column will be 100px, because we show column names
> entirely by default.
>
> [image: 2021-04-08_19h28_43]
> <https://user-images.githubusercontent.com/45152336/114062991-ce675580-98a0-11eb-8b61-fa2c19193877.gif>
>
> So, again, we show column names entirely by default, and we have some
> minimum value from column content, but we do not show column content
> entirely by default. An if you want to see entirely column names as well as
> column content - use Columns width: fit values.
>
> But if it does not work (and the setting is disabled, which I mentioned
> above) - this is bug. And I need to know more details about it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-815976198>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS5X5BUD45DTZFYHEB6LPFTTHXMO3ANCNFSM4YD5LQKQ>
> .
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by armandoclasslife** [comment]
I am using it on Linux.

On Thu, 8 Apr 2021 at 18:50, Armando Loscos ***@***.***>
wrote:

> as you can see on the screenshot, this is not true.
>
> selecting fit values, you don't see the entire column names
>
> regards
>
> On Thu, 8 Apr 2021 at 18:44, Anastasiya ***@***.***> wrote:
>
>> I not sure, that I understand you correctly.
>>
>> We have some standards for columns width. On default we use *fit screen*
>> - that you can see, when open table.
>>
>> You can see all content on default columns in case, when this is some
>> text column, when you are using *fit values*. If we have column name 50
>> px and content 100px - this column will be 100px. If we have column name
>> 100px and content 50px - this column will be 100px, because we show column
>> names entirely by default.
>>
>> [image: 2021-04-08_19h28_43]
>> <https://user-images.githubusercontent.com/45152336/114062991-ce675580-98a0-11eb-8b61-fa2c19193877.gif>
>>
>> So, again, we show column names entirely by default, and we have some
>> minimum value from column content, but we do not show column content
>> entirely by default. An if you want to see entirely column names as well as
>> column content - use Columns width: fit values.
>>
>> But if it does not work (and the setting is disabled, which I mentioned
>> above) - this is bug. And I need to know more details about it.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-815976198>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AS5X5BUD45DTZFYHEB6LPFTTHXMO3ANCNFSM4YD5LQKQ>
>> .
>>
>
>
> --
> *Armando Loscos*
> Classlife Education
> ***@***.***
> www.classlife.education
> (+34) 935 436 174
> ​
>
>
> Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
> contener información confidencial y sujeta al secreto profesional, cuya
> divulgación no está permitida por Ley. En caso de haber recibido este
> mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
> mediante correo electrónico remitido a nuestra atención y proceda a su
> eliminación, así como a la de cualquier documento adjunto al mismo.
>
> De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
> de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
> Personales y garantía de los derechos digitales, puede ejercer de forma
> totalmente gratuita los derechos de acceso, información, rectificación,
> supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
> no ser objeto de decisiones individuales automatizadas enviando un e-mail a
> ***@***.***
>
> Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
> <https://www.classlife.education/aviso-legal/> | Política de privacidad
> <https://www.classlife.education/privacidad/>.
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by armandoclasslife** [comment]
so it fits to the content but not to the max(content,column)

On Thu, 8 Apr 2021 at 18:50, Armando Loscos ***@***.***>
wrote:

> I am using it on Linux.
>
> On Thu, 8 Apr 2021 at 18:50, Armando Loscos ***@***.***>
> wrote:
>
>> as you can see on the screenshot, this is not true.
>>
>> selecting fit values, you don't see the entire column names
>>
>> regards
>>
>> On Thu, 8 Apr 2021 at 18:44, Anastasiya ***@***.***> wrote:
>>
>>> I not sure, that I understand you correctly.
>>>
>>> We have some standards for columns width. On default we use *fit screen*
>>> - that you can see, when open table.
>>>
>>> You can see all content on default columns in case, when this is some
>>> text column, when you are using *fit values*. If we have column name 50
>>> px and content 100px - this column will be 100px. If we have column name
>>> 100px and content 50px - this column will be 100px, because we show column
>>> names entirely by default.
>>>
>>> [image: 2021-04-08_19h28_43]
>>> <https://user-images.githubusercontent.com/45152336/114062991-ce675580-98a0-11eb-8b61-fa2c19193877.gif>
>>>
>>> So, again, we show column names entirely by default, and we have some
>>> minimum value from column content, but we do not show column content
>>> entirely by default. An if you want to see entirely column names as well as
>>> column content - use Columns width: fit values.
>>>
>>> But if it does not work (and the setting is disabled, which I mentioned
>>> above) - this is bug. And I need to know more details about it.
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-815976198>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AS5X5BUD45DTZFYHEB6LPFTTHXMO3ANCNFSM4YD5LQKQ>
>>> .
>>>
>>
>>
>> --
>> *Armando Loscos*
>> Classlife Education
>> ***@***.***
>> www.classlife.education
>> (+34) 935 436 174
>> ​
>>
>>
>> Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
>> contener información confidencial y sujeta al secreto profesional, cuya
>> divulgación no está permitida por Ley. En caso de haber recibido este
>> mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
>> mediante correo electrónico remitido a nuestra atención y proceda a su
>> eliminación, así como a la de cualquier documento adjunto al mismo.
>>
>> De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
>> de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
>> Personales y garantía de los derechos digitales, puede ejercer de forma
>> totalmente gratuita los derechos de acceso, información, rectificación,
>> supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
>> no ser objeto de decisiones individuales automatizadas enviando un e-mail a
>> ***@***.***
>>
>> Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
>> <https://www.classlife.education/aviso-legal/> | Política de privacidad
>> <https://www.classlife.education/privacidad/>.
>>
>
>
> --
> *Armando Loscos*
> Classlife Education
> ***@***.***
> www.classlife.education
> (+34) 935 436 174
> ​
>
>
> Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
> contener información confidencial y sujeta al secreto profesional, cuya
> divulgación no está permitida por Ley. En caso de haber recibido este
> mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
> mediante correo electrónico remitido a nuestra atención y proceda a su
> eliminación, así como a la de cualquier documento adjunto al mismo.
>
> De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
> de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
> Personales y garantía de los derechos digitales, puede ejercer de forma
> totalmente gratuita los derechos de acceso, información, rectificación,
> supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
> no ser objeto de decisiones individuales automatizadas enviando un e-mail a
> ***@***.***
>
> Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
> <https://www.classlife.education/aviso-legal/> | Política de privacidad
> <https://www.classlife.education/privacidad/>.
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by LonwoLonwo** [comment]
> as you can see on the screenshot

Where can I see screenshot?

**Comment by armandoclasslife** [comment]
sorry,now

On Thu, 8 Apr 2021 at 20:58, Anastasiya ***@***.***> wrote:

> as you can see on the screenshot
>
> Where can I see screenshot?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-816066816>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS5X5BVJECMUXKNVVFNZQL3THX4G3ANCNFSM4YD5LQKQ>
> .
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by armandoclasslife** [comment]
the fact the screenshot was there, but on the 1st out of the 3 emails. but
passed again.

On Fri, 9 Apr 2021 at 07:22, Armando Loscos ***@***.***>
wrote:

> sorry,now
>
> On Thu, 8 Apr 2021 at 20:58, Anastasiya ***@***.***> wrote:
>
>> as you can see on the screenshot
>>
>> Where can I see screenshot?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-816066816>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AS5X5BVJECMUXKNVVFNZQL3THX4G3ANCNFSM4YD5LQKQ>
>> .
>>
>
>
> --
> *Armando Loscos*
> Classlife Education
> ***@***.***
> www.classlife.education
> (+34) 935 436 174
> ​
>
>
> Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
> contener información confidencial y sujeta al secreto profesional, cuya
> divulgación no está permitida por Ley. En caso de haber recibido este
> mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
> mediante correo electrónico remitido a nuestra atención y proceda a su
> eliminación, así como a la de cualquier documento adjunto al mismo.
>
> De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
> de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
> Personales y garantía de los derechos digitales, puede ejercer de forma
> totalmente gratuita los derechos de acceso, información, rectificación,
> supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
> no ser objeto de decisiones individuales automatizadas enviando un e-mail a
> ***@***.***
>
> Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
> <https://www.classlife.education/aviso-legal/> | Política de privacidad
> <https://www.classlife.education/privacidad/>.
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by p91paul** [comment]
@armandoclasslife screenshots or other files attached to emails are not supported by github, you have to reply through website to attach files.

**Comment by armandoclasslife** [comment]
adding screenshot here.thanks p91paul
![dbeaver](https://user-images.githubusercontent.com/79396486/114158464-fe583c80-9924-11eb-9dc5-3c2e93f6b141.png)

**Comment by LonwoLonwo** [comment]
Can you please check datasource settings?

Please click "Configure" bottom in the lower part of grid in Data viewer.
And then attach here photo of "Appearance" tab from Editors->Data Editor->Appearance

![2021-04-12 10_56_33-DBeaver 21 0 1 - film](https://user-images.githubusercontent.com/45152336/114360643-f0e9bf00-9b7d-11eb-9a20-7a0966d2335d.png)

![2021-04-12 10_56_59-Properties for](https://user-images.githubusercontent.com/45152336/114361057-6a81ad00-9b7e-11eb-9986-38ad9a3d0759.png)

**Comment by armandoclasslife** [comment]
HI,

not necessary, I found what you mean, that is, it is necessary to disable:

calculate column width...


Thanks a million!

On Mon, 12 Apr 2021 at 10:01, Anastasiya ***@***.***> wrote:

> Can you please check datasource settings?
>
> Please click "Configure" bottom in the lower part of grid in Data viewer.
> And then attach here photo of "Appearance" tab from Editors->Data
> Editor->Appearance
>
> [image: 2021-04-12 10_56_33-DBeaver 21 0 1 - film]
> <https://user-images.githubusercontent.com/45152336/114360643-f0e9bf00-9b7d-11eb-9a20-7a0966d2335d.png>
>
> [image: 2021-04-12 10_56_59-Properties for]
> <https://user-images.githubusercontent.com/45152336/114361057-6a81ad00-9b7e-11eb-9986-38ad9a3d0759.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dbeaver/dbeaver/issues/11461#issuecomment-817583444>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS5X5BRSZEX5YHJEEAPKN53TIKSGXANCNFSM4YD5LQKQ>
> .
>


-- 
*Armando Loscos*
Classlife Education
***@***.***
www.classlife.education
(+34) 935 436 174
​


Este mensaje va dirigido, de manera exclusiva, a su destinatario y puede
contener información confidencial y sujeta al secreto profesional, cuya
divulgación no está permitida por Ley. En caso de haber recibido este
mensaje por error, le rogamos que, de forma inmediata, nos lo comunique
mediante correo electrónico remitido a nuestra atención y proceda a su
eliminación, así como a la de cualquier documento adjunto al mismo.

De conformidad con el Reglamento (UE) núm.2016/679, General de Protección
de Datos y la LO 3/2018, de 5 de diciembre, de Protección de Datos
Personales y garantía de los derechos digitales, puede ejercer de forma
totalmente gratuita los derechos de acceso, información, rectificación,
supresión y olvido, limitación del tratamiento, oposición, portabilidad y a
no ser objeto de decisiones individuales automatizadas enviando un e-mail a
***@***.***

Responsable: Classlife Education, S.L. | NIF: B-66490962 | Aviso legal
<https://www.classlife.education/aviso-legal/> | Política de privacidad
<https://www.classlife.education/privacidad/>.

**Comment by LonwoLonwo** [comment]
@armandoclasslife hooray!
